unlock-app/src/components/creator/CreatorLockForm.js

```diff
@@ -131,9 +127,12 @@ const CreatorLockForm = ({ hideAction, lock, saveLock }) => {
         }
         break
       case 'maxNumberOfKeys':
-        if (value !== UNLIMITED_KEYS_COUNT && !isPositiveInteger(value)) {
+        if (!isPositiveInteger(value)) {
```

Comment:
Actually I think we should let creators deploy locks with 0 keys since they can either change it later (useful when someone wants to deploy everything and test things without "selling" memberships right away) or even only ever "grant" keys to anyone.